src/components/Stories.js

```diff
@@ -4,12 +4,15 @@ export default (props) => {
   return(
     <div>
       <h1>Stories</h1>
-      {props.stories.map((story) => {
+      {props.stories.map((story, index) => {
           return(
-            <ul>
+            <ul key={index}>
               <div className="card card-panel">
+                <li>ID: {story.id}</li>
                 <li>{story.translated_content}</li>
                 <li>By: {story.creator}</li>
+                <a className="btn" href="#translatorBox" onClick={function(){props.edit(story.id)}}> Edit </a>
```

Comment:
i would use arrow functions here - your choice but i think it reads easier
`function(){props.edit(story.id)}` vs `() => props.edit(story.id)`